src/include/simeng/arch/riscv/Instruction.hh

```diff
@@ -169,6 +169,7 @@ class Instruction : public simeng::Instruction {
   /** Retrieve the instruction's associated architecture. */
   const Architecture& getArchitecture() const;
 
+ private:
```

Comment:
Seem like Private member region already defined above unless GitHub is being odd

Reply:
Good spot, removed